plugins/router/f5/f5.go

```diff
@@ -723,7 +841,7 @@ func (f5 *f5LTM) CreatePool(poolname string) error {
 
 	payload := f5Pool{
 		Mode:    "round-robin",
-		Monitor: "/Common/http",
+		Monitor: fmt.Sprintf("%s/http", f5.partitionPath),
```

Comment:
I'm not sure this is correct.  We might still want to use `/Common/http` as the default monitor (in the future, we should allow the administrator to specify a different monitor to use, but that's a separate matter) because otherwise, I would think we would need to create the `http` monitor under the configured partition; is that not so?